ui-v2/app/templates/dc/services/-exposedpaths.hbs

```diff
@@ -0,0 +1,33 @@
+<p>
+	You can expose individual HTTP paths like /metrics through Envoy for external services like Prometheus.
+</p>
+{{#tabular-collection
+    data-test-exposedpaths
+    class="exposedpaths"
+    items=item.Proxy.Expose.Paths as |path index|
+}}
+    {{#block-slot 'header'}}
+      <th>Path</th>
+      <th>Protocol</th>
+      <th>Listener port</th>
+      <th>Local path port</th>
+      <th>Combined address<span><em role="tooltip">Service address, listener port, and path all combined into one URL.</em></span></th>
+    {{/block-slot}}
+    {{#block-slot 'row'}}
+      <td>
+        <span>{{path.Path}}</span>
+      </td>
+      <td>
+        {{path.Protocol}}
+      </td>
+      <td>
+        {{path.ListenerPort}}
+      </td>
+      <td>
+        {{path.LocalPathPort}}
+      </td>
+      <td>
+        <span data-test-combined-address>{{item.Address}}:{{path.ListenerPort}}{{path.Path}}</span>
```

Comment:
Do we want to add our copy link here on hover of the row? That's how I intended it, but totally understand if it's not MVP.

Reply:
Lets see where we get with the disco chain / namespace work. We can potentially come back here and add these. Folks can still just use their mouse to select exactly what they want to copy, whether it's the entire text, or just the path or part of the path, so for me it wouldn't be MVP, but happy to add once we get this in.

I think the other thing I'm conscious of here is horizontal space in the rows. Thinking out loud a little, it might be something we could add as standard across the entire app once we move to Composite Rows. If we do that we can say 'every top row of the Composite Row should be copyable via a button' then people grow to expect that and its consistent through the app.